esmvaltool/namelist.py

```diff
@@ -192,13 +192,19 @@ def _get_value(key, variables):
 def _add_cmor_info(variable, keys):
```

Comment:
This was just to use variable['project'] as a default for variable['cmor_table']  to avoid having to specify it two times in most cases. But I think that the correct place to do this is at namelist reading level, so I will change it

Reply:
Done